<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#Form
{
	background-color: rgb(213, 213, 213);
}


QWidget
{
	background-color: rgb(239, 239, 239);
}


QPushButton {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(234, 234, 234, 255), 
stop:0.5 rgba(249, 249, 249, 255),
stop:1 rgba(234, 234, 234, 255));
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255)
 }

 QPushButton:pressed {
	background-color: rgb(221, 221, 221);
 }

 QPushButton:hover {
	 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(229, 229, 229, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(229, 229, 229, 221));
    border-color: rgba(221, 128, 32, 255)
 }


QLineEdit {
	background-color:  rgba(255, 255, 255, 255);
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255)
 }
</string>
  </property>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>180</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/resource/logo.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>445</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_pop_cat" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>180</width>
     <height>240</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_pop_cat">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>7</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Popular Categories</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>75</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>95</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>115</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>135</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>155</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_8">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>175</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_9">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>195</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat_10">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>215</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_login" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>295</y>
     <width>180</width>
     <height>145</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_user_head">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_pass_head">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>8</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>8</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_seebidhistory">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>515</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>See Bid History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sell">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>405</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_seeprofile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>See Profile</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_info" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>295</y>
     <width>180</width>
     <height>105</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>27</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>[name]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>[username]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_userid">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>80</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>[userid]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>7</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome back,</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_prev">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>590</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>590</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_p1">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>590</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>[1]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_p2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>590</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>[2]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_p3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>590</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>[3]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_p4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>590</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>[4]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_prev_6">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>590</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>[5]</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_6" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_2" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_7" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_9" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_8" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_3" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_4" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_5" native="true">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_unused_10" native="true">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>320</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
